### PR TITLE
refactor(services): implementing FQDN for all services

### PR DESCRIPTION
**Pull request by NeilMasters**

### CODE REVIEW

File: Resources/config/services.yml [hunk 1]
```diff
@@ -30,63 +38,99 @@ services:
...
+    Markup\JobQueueBundle\Service\SupervisordConfigFileWriter: '@markup_job_queue.writer.supervisord_config_file'
+
     markup_job_queue.scheduled:
         class: Markup\JobQueueBundle\Service\ScheduledJobService
         arguments:
             - '@doctrine'
 
+    Markup\JobQueueBundle\Service\ScheduledJobService: '@markup_job_queue.scheduled'
+
     markup_job_queue.writer.cli_consumer_config_file:
```
[the remark this answers is not in the document]
Comment (shieldo) [reply Resources/config/services.yml]: Certainly my opinion is that no service should be being instantiated during container building...

[the remark this answers is not in the document]
Comment (NeilMasters) [reply Resources/config/services.yml]: Apologies I communicated that incorrectly. The class itself is being instantiated when the services.yml is built however its constructor arguments are only ever being set by the extension:

https://github.com/usemarkup/JobQueueBundle/blob/d5d26c257a02c55f95be6bcc67ed9e8b8488b617/DependencyInjection/MarkupJobQueueExtension.php#L62 replacing the args.

I do not think it needs to be a pre-defined service - if we are injecting the constructor args we might as well make a new definition at that time?

[the remark this answers is not in the document]
Comment (shieldo) [reply Resources/config/services.yml]: Which deprecation specifically is Symfony whinging about here?

[the remark this answers is not in the document]
Comment (shieldo) [reply Resources/config/services.yml]: @NeilMasters Which is the deprecation referred to here as one being whinged about?

[the remark this answers is not in the document]
Comment (NeilMasters) [reply Resources/config/services.yml]: Sorry @shieldo - this is the deprecation we discussed in person and I _was_ meant to add. We agreed that I need to add an abstract: true onto the definition to get rid of the notice.

**Comment by NeilMasters** [comment]
Closing this PR. It is sensible change but it muddies up my own PR queue everytime i see it and go 'huh? whats that?'.

Reopen if anyone wants it merged.

**Pull request closed without merging**